ci/bin/concourse_job_performance.py

```diff
@@ -58,7 +60,7 @@ def main(url, team, pipeline, job, number_of_builds, authorization_cookie, threa
     builds = get_builds_summary_sheet(url, team, pipeline, job, number_of_builds+10, authorization_cookie)
 
     build_to_examine = get_builds_to_examine(builds, number_of_builds)
-    expected_failed_builds = [int(b['name']) for b in build_to_examine if b['status'] == 'failed']
+    expected_failed_builds = [int(b['name']) for b in build_to_examine if b['status'] in ['failed', 'succeeded']]
```

Comment:
You might want to rename the variable if now you are looking at failed and succeeded builds.

Reply:
done